controller/rancher/rancher.go

```diff
@@ -492,6 +526,14 @@ func (lbc *LoadBalancerController) processSelector(lbMeta *LBMetadata) error {
 			continue
 		}
 
+		regionName := lbRule.Region
+		envName := lbRule.Environment
+
+		svcs, err := lbc.GetRegionServices(&svcs, regionName, envName)
+		if err != nil {
+			logrus.Warnf("couldn't find services from region %v", err)
```

Comment:
should we return on error?

Reply:
There could be other PortRules that we might need to consider, so can't return?  (will add a continue instead though)

Reply:
@kinarashah continue